ListUtil.xs

```diff
@@ -1200,7 +1200,7 @@ CODE:
         SvIsUV_on(TARG);
     }
 #endif
-    else {
+    if(SvIOK(num)) {
```

Comment:
If this `else` is going to disappear, I feel that the `else` on line 1197 above ought to too.